.kokoro/release/common.cfg

```diff
@@ -38,12 +38,3 @@ env_vars: {
   key: "SECRET_MANAGER_KEYS"
   value: "releasetool-publish-reporter-app,releasetool-publish-reporter-googleapis-installation,releasetool-publish-reporter-pem"
 }
-
-# Store the packages we uploaded to PyPI.  That way, we have a record of exactly
```

Comment:
Is this removal intentional? I'm asking because the comment about SBOMs and attestations seems important.

Reply:
It looks like this change was from a commit made by owlbot (https://github.com/googleapis/gapic-generator-python/pull/1652/commits/b2f902db1f4facce01bfa68627735b976bb7fcc5), which was followed by another commit from owlbot (https://github.com/googleapis/gapic-generator-python/pull/1652/commits/3d78cb8396e86bc1db4f4d20049b35451b9ca5a3) which reverted the change. Closing as obsolete.